.readthedocs.yaml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
+
+sphinx:
+   configuration: docs/conf.py
+
+formats:
+  - epub
+  - pdf
+
+python:
+   install:
+   - requirements: requirements.txt
```

Comment:
Do you think should we replace it to environment.yml?

Reply:
I would prefer to use pip-dependencies file to correspond with default settings from readthedocs GUI. Besides all doc building is happened without conda involvement.